### ISSUE TITLE
cannot install python packages in the start task

### ISSUE THREAD

**Issue by sbirmole211038** [Azure/Batch#146]
hey guys, i am currently facing an issue where i have built an etl pipeline in azure data factory where i use blob trigger to activate the custom task on  batch account, i have set up start task for the nodes where i install python and pip and other libraries but python and pip gets installed however the packages don't  and the code throws an error saying package missing i have tried using requirements.txt and also whl files for the packages but got no luck, can anyone help me with this issue, thank you in advance
![image](https://user-images.githubusercontent.com/113025543/212137744-21ae22bb-3336-41f5-a44c-d6f11b38792b.png)
the image above is an screenshot of start task command 
i have checked that pip and python gets installed by logging in the vd(virtual desktop) and trying commands in the cmd however the python package don't get installed, and even after installing them in the vd as per my knowledge it creates new instance every time the job runs hence i cant run the code even if  i install it in the vd it shows the same error i am guessing it uses new instance every time

**Comment by alfpark** [comment]
I don't think this is a valid start task. You're trying to run multiple `cmd /c` executions. You either need to chain these together in one `cmd /c` command or use a windows command/batch script.
